NMRangeSlider/Slider.xcassets/slider-default7-trackBackground.imageset/Contents.json

```diff
@@ -6,7 +6,6 @@
     },
     {
```

Comment:
Shouldn't we remove the whole object? Now only the file name property is removed :-)